scripts/import.py

```diff
@@ -23,13 +25,21 @@ def list_files(directory):
     print("Access token: {}".format(access_token))
     print("Listing dir: {}".format(listing_dir))
 
-    for file_path in list_files(listing_dir):
+    def put_file(file_path):
         with open(file_path) as f:
-            response = requests.post(
-                endpoint,
-                data='{"services":%s}' % f.read(),
+            data = json.load(f)
+            data = {'services': data}
+            url = '{}/{}'.format(endpoint, data['services']['id'])
+            response = requests.put(
+                url,
+                data=json.dumps(data),
                 headers={
                     "content-type": "application/json",
                     "authorization": "Bearer {}".format(access_token),
                 })
-            print(response)
+            return response
+
+    pool = multiprocessing.Pool(10)
+
+    for result in pool.imap(put_file, list_files(listing_dir)):
+        print(result)
```

Comment:
In the case of error responses (4xx, 5xx) it would be useful to print the filename of the failed upload.